src/lib/components/linetimeseriechart/linetimeseriechart.component.tsx

```diff
@@ -0,0 +1,549 @@
+  margin-right: 8px;
+  flex-shrink: 0;
+  margin-top: 8px;
+`;
+
+const TooltipContent = styled.div`
+  display: flex;
+  min-width: 0;
+  flex: 1;
+`;
+
+const TooltipName = styled.div`
+  margin-right: 4px;
+  word-wrap: break-word;
+  word-break: break-word;
+  justify-content: flex-start;
+`;
+
+const TooltipInstanceValue = styled.div`
+  justify-content: flex-end;
+`;
+
+export type Serie = {
+  // the name of the resource
+  resource: string;
+  // the original data format from prometheus
+  data: [number, string | null][];
+  // it's mandatory to display tooltip label in the tooltip
+  getTooltipLabel: (metricPrefix?: string, resource?: string) => string;
+  // get the legend label for each of the series
+  getLegendLabel?: (metricPrefix?: string, resource?: string) => string;
+  // optional color field to specify the color of the line
+  color?: string;
+  // the name of the metric prefix with read, write, in, out
+  metricPrefix?: string;
+  // to specify if the line is dash
+  isLineDashed?: boolean;
+};
+
+type NonSymmetricalChartSerie = {
+  yAxisType?: 'default' | 'percentage';
+  series: Serie[];
+};
+
+// The symmetrical chart props are used to display two series on the same chart, such as in/out, write/read
+type SymmetricalChartSerie = {
+  yAxisType: 'symmetrical';
+  series: {
+    above: Serie[];
+    below: Serie[];
+  };
+};
+
+export type LineChartProps = (
+  | NonSymmetricalChartSerie
+  | SymmetricalChartSerie
+) & {
+  title: string;
+  height: number;
+  startingTimeStamp: number;
+  unitRange?: {
+    threshold: number;
+    label: string;
+  }[];
+  isLoading?: boolean;
+  isLegendHidden?: boolean;
+  yAxisTitle?: string;
+  helpText?: string;
+  onHover?: (payload: any) => void;
+  lineColor?: string;
+};
+
+const CustomTooltip = ({
+  active,
+  payload,
+  label,
+  unitLabel,
+}: {
+  active?: boolean;
+  payload?: Array<{
+    value: number;
+    name: string;
+    color: string;
+    dataKey: string;
+  }>;
+  label?: string;
+  unitLabel?: string;
+}) => {
+  if (!active || !payload || !payload.length || !label) return null;
+  // We can't use the default itemSorter method because it's a custom tooltip.
+  // Sort the payload here instead
+  const sortedPayload = [...payload].sort((a, b) => {
+    const aValue = Number(a.value);
+    const bValue = Number(b.value);
+
+    if (aValue >= 0 && bValue >= 0) {
+      return bValue - aValue; // Higher positive values first
+    }
+    if (aValue < 0 && bValue < 0) {
+      return bValue - aValue; // Lower negative values first
+    }
+    return bValue - aValue; // Positives before negatives
+  });
+
+  return (
+    <TooltipContainer>
+      <TooltipTime>
+        {new Date(label)
+          .toLocaleString('en-GB', {
+            day: 'numeric',
+            month: 'short',
+            hour: '2-digit',
+            minute: '2-digit',
+            second: '2-digit',
+            hour12: false,
+          })
+          .replace(',', '')}
```

Comment:
I think this can be replaced by the DateFormater

Reply:
Yes I've created a new formate since we don't have this before.